heron/api/src/java/BUILD

```diff
@@ -11,23 +12,37 @@ api_deps_files =  \
         "//third_party/java:kryo",
     ]
 
+# Kryo dependency should be specified in POM
+api_deps_files_nl =  \
+    heron_java_proto_files() + [
+        ":classification",
+        "//heron/common/src/java:basics-java",
+        "//third_party/java:kryo-neverlink",
+    ]
+
 java_library(
     name = "api-java",
     srcs = glob(["com/twitter/heron/api/**/*.java"]),
     deps = api_deps_files,
 )
 
+java_library(
```

Comment:
Could we only use the neverlink version and then add kryo as it's own jar to whatever artifact bundle should also have it?